### PR TITLE
feedback

### PR DESCRIPTION
**Pull request by bhandariabhishek457**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ACCESS_TOKEN_SECRET = jsfgfjguwrg8783wgbjs849h2fu3cnsvh8wyr8fhwfvi2g225
 REFRESH_TOKEN_SECRET = 825y8i3hnfjmsbv7gwajbl7fobqrjfvbs7gbfj2q3bgh8f42
 
-TEST_TOKEN= eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjE4LCJlbWFpbCI6ImFAZ21haWwuY29tIiwiaWF0IjoxNjcwNzU1MzM2LCJleHAiOjE2NzA3NTYyMzZ9.RUu9kwa9sJusJV9c6bN8xGOuDmOBLvOeAEOp5xx0n2c
+TEST_TOKEN= eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjIyLCJlbWFpbCI6ImFiQGdtYWlsLmNvbSIsImlhdCI6MTY3MDgzNzM1MywiZXhwIjoxNjcwOTIzNzUzfQ.ICxWakfKoTgbB4QJy703vU4JBzSXVuh7gVZmNybbfTs
 
 port = 5000
 DB = lab1
```
Comment (ShishirBhandari) [inline .env]: .env file should not be added to the git.
You should ignore this file by adding it to the `.gitignore` file.

Comment (ShishirBhandari) [reply .env]: because it contains secret information

Comment (bhandariabhishek457) [reply .env]: Added to gitignore

File: src/services/loginService.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	});
+};
+
+export const findAllUserByRefreshToken = async (refreshToken) => {
+	return await Users.findAll({
+		where: {
+			refresh_token: refreshToken,
+		},
+	});
+};
```
Comment (ShishirBhandari) [inline src/services/loginService.js]: We don't need findall because refresh token is each user has unique token

Comment (bhandariabhishek457) [reply src/services/loginService.js]: Fixed

**Comment by ShishirBhandari** [comment]
Also, please add node_modules folder to `.gitignore`.

We should not include package folder to the github.

**Pull request merged**